src/handler.php

```diff
@@ -163,13 +168,13 @@ public static function noDupes(array &$events) {
             if ($events[$i - 1]->dtstart === $events[$i]->dtstart
                 && $events[$i - 1]->dtend === $events[$i]->dtend
                 && $events[$i - 1]->summary === $events[$i]->summary) {
-                //if this and next event are the same, check whether the next is held in interims
-                if (strpos($events[$i]->location, "Interims")) {
-                    //Append the location from the next (same) element to this element
-                    $events[$i - 1]->location .= "\n" . $events[$i]->location;
-                    //Mark next element for removal
-                    unset($events[$i]);
-                    $i--;
```

Comment:
Can we just keep both of them?

Reply:
Keeping both would mean not removing the duplicate. The only way that could be done I guess is if events can be set in iCal to be hidden. I just took inspiration from the way the function was already written

Reply:
I mean changing the condition to `strpos($events[$i]->location, "Interims") || strpos($events[$i]->description, "Videoübertragung")`

Reply:
From my understanding, this change was originally made in order to always prefer the live lecture to a video transmission of the same (see #30) since video transmissions tend to take place in interims. The condition I changed still covers those cases. Adding interims to the condition would mean that any location would be preferred to interims in any case, which I think misses the point (think of a situation where the live lecture is in interims and the video transmission goes to the city center campus for example).